homeassistant/components/flux_led/light.py

```diff
@@ -140,6 +141,11 @@
 )
 
 
+def flux_led_wait():
+    """Wait device after change state to improve stability."""
+    time.sleep(1)
```

Comment:
We shouldn't sleep in threads. We should only sleep with `await asyncio.sleep(1)` because it doesn't actually block anything.

Reply:
Totally agree, I will turn the whole module async then.